extensions/azurecore/src/azureResource/providers/database/databaseService.ts

```diff
@@ -35,7 +35,7 @@ export class AzureResourceDatabaseService implements IAzureResourceService<azure
 		});
 
 		// Match database ID. When calling exec [0] is full match, [1] is resource group name, [2] is server name
-		const svrIdRegExp = new RegExp(`\/subscriptions\/${subscription.id}\/resourceGroups\/(.+)\/providers\/Microsoft\.Sql\/servers\/(.+)\/databases\/.+`);
+		const svrIdRegExp = new RegExp(`\/subscriptions\/.+\/resourceGroups\/(.+)\/providers\/Microsoft\.Sql\/servers\/(.+)\/databases\/.+`);
```

Comment:
I don't see why having the id in here is necessary - the responses should all be from the expected subs. It wouldn't make sense to have anything else.